### ISSUE TITLE
Improving ggforest

### ISSUE THREAD

**Issue by kassambara** [kassambara/survminer#178]
I just discovered the forestmodel package (https://github.com/NikNakk/forestmodel), which creates forest plots of the coefficients of models produced by lm, glm, survival::coxph

```r
library("forestmodel")
library("survival")
library("dplyr")
pretty_lung <- lung %>%
  transmute(time,
            status,
            Age = age,
            Sex = factor(sex, labels = c("Male", "Female")),
            ECOG = factor(lung$ph.ecog),
            `Meal Cal` = meal.cal)

print(forest_model(coxph(Surv(time, status) ~ ., pretty_lung)))
```

![rplot01](https://cloud.githubusercontent.com/assets/3313355/24242348/c4ca12e0-0fb7-11e7-8d1a-79c8c32799b9.png)

@pbiecek : In ggforest(), I vote for staying focused on survival::coxph. But, how can we improve the output ? shoud we use internally the forestmodel package?

**Comment by pbiecek** [comment]
Cool, it works for lm and glm as well.
Will look into this.

**Comment by pbiecek** [comment]
It took me some time to look for some other ideas for forest model plots that will be as nice as the one generated by NikNakk. Will submit a new version of `ggforest` this weekend.

**Comment by kassambara** [comment]
Ok, great!!!

**Comment by pbiecek** [comment]
The new version of ggforest looks like the figure below (will be uploaded soon, need some polishing)
I borrowed some ideas from forestmodel package. In addition to standard elements in the bottom-left corner there are some model statistics (AIC/n.events and others). 
<img width="739" alt="forest2" src="https://cloud.githubusercontent.com/assets/4624318/24582948/437674a4-173c-11e7-839e-9245c34f996a.png">

**Comment by kassambara** [comment]
Thank you @pbiecek for this great work!

Default text size in the `ggforest()` script is 0.7,  some users might want to change this.  
Suggestion: additional argument fontsize or labelsize or ...

I observed that, the hazard ratio axis ticks are not displayed sometimes. It seems to depend on the model.  Do you have any idea? 

For example,

1) In the example below, the axis ticks are displayed:

```r
require("survival")
require("survminer")
model <- coxph( Surv(time, status) ~ sex+rx,
                data = colon )
ggforest(model)
```

![rplot](https://cloud.githubusercontent.com/assets/3313355/24585111/9e292838-1782-11e7-95a5-6b7386ea2b40.png)

2) But, in this second example, the ticks are not shown:

```r
require("survival")
require("survminer")
model <- coxph( Surv(time, status) ~ sex+rx+adhere,
                data = colon )
ggforest(model)
```

![rplot04](https://cloud.githubusercontent.com/assets/3313355/24585119/e9681fa2-1782-11e7-8e99-d1fb20409a38.png)

**Comment by pbiecek** [comment]
Ticks are not displayed because the scale vary from 0.55 to 1.59 and the 'pretty printing' cannot find nice looking ticks in such narrow range.
One solution is to force the OX scale to be at least c(0.5, 2), 
will add this if there are no other candidate solutions.

Having more graphical parameters exposed is a good idea, will do this.
However, the left panel is not a pure ggplot2 plot, so we cannot simply use `theme` parameter.

**Comment by pbiecek** [comment]
Ok, things are even stranger, since in the first plot the OX scale is more narrow.
Will look into this.

**Comment by pbiecek** [comment]
the problem was in using scale_x limits instead of coord_ limits
fixed in #182

**Comment by rosestevens** [comment]
@pbiecek Hi I am using ggforest to create a figure for a paper I'm working on. I need to be able to make the font Times New Roman for the journal formatting. Is there any way to do this with ggforest? Happy to send over a reproducible example if it would help. Thanks so much, Rose.
